knpu/start-project.md

```diff
@@ -25,34 +25,65 @@ doing *amazing* things with the Symfony framework.
 Let's get our first Symfony project rolling. Head over to Symfony.com and click
```

Comment:
@weaverryan Should we make `Symfony.com` a link here? I do it for internal... but I'm not sure for external links.